xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
 
+<a name="Introduction"/>
+## 1.0 Introduction
 
-<a name="roleofxapi"/>
-## 1.0 Summary of the Experience API
+The Experience API (xAPI) is a technical specification that aims to facilitate the documentation and communication of learning experiences.  It does so by setting out a protocol that specifies how learning experiences should be described and an Application Programming Interface that sets out how information about these experiences should be stored and exchanged electronically.  
 
-The Experience API (xAPI) is an Application Programming Interface (API) specification that 
-facilitates communication about experiences between a Learning Record Store and a client, 
-typically an Activity Provider. This specification was developed with learning experiences in mind, 
-but many other kinds of experiences could be tracked using xAPI. 
+The xAPI is part of the Advanced Distributed Learning (ADL) initiative, established in 1997 to standardize and modernize training and education management and delivery in the US. Among the many outcomes of this initiative was SCORM - the Shareable Content Object Reference Model. SCORM has 
+become a great success and an internationally recongized framework that has greatly contributed greatly to the economy and effectiveness of online learning.
+  
+While SCORM has been a great success, there has been a growing realization that there is a huge need and potential to track learning experiences of individuals beyond formal, structured computer-based traing.  In assessing candidates' suitability for positions or their capability form performing various tasks, there is a need to consider a wide range of formal and informal learning experiences, both on and offline.  That information, more often than not be scattered across a wide variety of sources.
 
-This specification does not attempt to standardize how the Activity Provider facilitates and tracks the learning experience, 
-nor how the Learning Store stores and procesess that data. The scope of this specification is limited to the requests and responses
-between Learning Record Store and client. 
+Out of this perceived need, the xAPI initiative and specification was born.  xAPI assumes that:
```

Comment:
For me, putting the term initiative in lower case suggests that we are not
talking about an organization.  I would like to leave initiative in because
a whole bunch more that the specification was was born from this need - not
the least of which is the communities of practice.

On Wed, Sep 30, 2015 at 11:11 AM, Andrew Downes <notifications@github.com>
wrote:

> In xAPI.md
> <https://github.com/adlnet/xAPI-Spec/pull/759#discussion_r40829424>:
>
> >
> > -This specification does not attempt to standardize how the Activity Provider facilitates and tracks the learning experience,
> > -nor how the Learning Store stores and procesess that data. The scope of this specification is limited to the requests and responses
> > -between Learning Record Store and client.
> > +Out of this perceived need, the xAPI initiative and specification was born.  xAPI assumes that:
>
> Sorry, I should have explained more. To me "xAPI Initiative" sounds like
> an organization, but no such organization exists so i think it could be
> confusing. We could just delete "initiative and "?
>
> I guess really you're referring to a continuum made up of Project Tin Can
> and the xAPI Working Group.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adlnet/xAPI-Spec/pull/759/files#r40829424>.
>



-- 
Roger Swetnam
p.  604-408-5703
c.  778-848-3118